### PR TITLE
OS X Support

### PR DESCRIPTION
**Pull request by odensc**
Worked on OS X before this, but the tray icon was black and glitched out.
Should also work on Linux (haven't tested), but someone needs to add builds to the readme for OS X and Linux.

### CODE REVIEW

**Comment by brennan-macaig** [comment]
Will test on Ubuntu-Linux 64Bit tomorrow when I get the chance.

**Comment by skibz** [comment]
i've not tested your pull request, however, the HEAD at master seems to work fine on my MBA with osx yosemite.

what does `iconsAreTemplates` do?

**Comment by odensc** [comment]
@skibz Prior to this PR, the tray icon looked like a black cube with half of the letter "T". The explanation is on the nw.js wiki:

Tray.iconsAreTemplates

since v0.11.3

Mac OS X Only. Get or Set whether icon and alticon images are treated as "templates" (true by default). When the property is set to true the images are treated as "templates" and the system automatically ensures proper styling according to the various states of the status item (e.g. dark menu, light menu, etc.). Template images should consist only of black and clear colours and can use the alpha channel in the image to adjust the opacity of black content. It has no effects on Linux and Windows. For further information see #2775.

Basically if it's true (which it is by default), it will convert the icon to be black or white depending on whether the top bar is on dark mode or white mode.

**Comment by skibz** [comment]
ah, very cool.

**Comment by CyrisXD** [comment]
Thanks @thesbros, please add yourself to the about.html contributors. We will need to work on compiling the linux/osx versions once SteamRoller is stable with the main features wanted.
